api/base/exceptions.py

```diff
@@ -83,6 +83,10 @@ def json_api_exception_handler(exc, context):
     return response
 
 
+class NotImplementedError(APIException):
```

Comment:
`EndpointNotImplementedError` or `APINotImplementedError` might be most appropriate. (Django has an `HTTPNotImplementedError`)